Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -20,6 +20,7 @@ public class CBPurchase: NSObject {
     private var restoredPurchasesCount = 0
     private var activeProduct: SKProduct?
     var customerID: String = ""
+    var customerInfo: CBCustomerInfo?
```

Comment:
Can we remove `customerID` and use the `CBCustomerInfo` instead?

Reply:
yes its possible here to pass customerID in CBCustomerInfo object. will update it.